opentelemetry-sdk/src/export/logs/mod.rs

```diff
@@ -0,0 +1,33 @@
+//! Log exporters
+use crate::Resource;
+use async_trait::async_trait;
+use opentelemetry_api::{
+    logs::{LogError, LogRecord, LogResult},
+    InstrumentationLibrary,
+};
+use std::{borrow::Cow, fmt::Debug};
+
+/// `LogExporter` defines the interface that log exporters should implement.
+#[async_trait]
```

Comment:
I don't think traits that every 3rd-party exporter is expected to implement should be forcing non-standard things like the async-trait crate. It's hacky and unnecessary, and will probably turn into an issue in the future when/if built-in support for async functions in traits shows up.

While the SpanExporter's BoxedFuture is unpleasant, it feels like a more future-proof design and one that is less burdensome on developers.

Reply:
Agree, exporter trait needn't be async, and let the processor decide how this exporter would be invoked. We can keep this consistent with SpanExporter - https://github.com/open-telemetry/opentelemetry-rust/blob/69fd972fa63a3cb75f61fa60ffdac210071b93f7/opentelemetry-sdk/src/export/trace.rs#L19